arviz/tests/base_tests/test_labels.py

```diff
@@ -0,0 +1,70 @@
+from pytest_cases import fixture_ref
+from ..labels import (BaseLabeller,
+                    DimCoordLabeller,
+                    IdxLabeller,
+                    DimIdxLabeller,
+                    MapLabeller,
+                    NoVarLabeller,
+                    NoModelLabeller,
+                    mix_labellers)
+
+@pytest.fixture(scope="module")
+class Data():
+    def __init__(self):
+            self.sel= {
+                "instrument": "a",
+                "experiment": 3,
+            }
+            self.isel= {
+                "instrument": 0,
+                "experiment": 4,
+            }
+
+class TestLabellers():
+    @pytest.fixture(scope="class")
+    def __init__(self):
+        # the MapLabeller should be initialized with some mappings on all levels
+        # if we decide to add NoRepeatLabeller as part of ArviZ it should not be
+        # tested here but have dedicated test functions due to its special functionality,
+        # for now simply skip it
+    
+        self.labellers = {"BaseLabeller": BaseLabeller(), 
+                          "DimCoordLabeller": DimCoordLabeller(),
+                          "IdxLabeller": IdxLabeller(),
+                          "DimIdxLabeller": DimIdxLabeller(),
+                          "MapLabeller": MapLabeller(),
+                          "NoVarLabeller": NoVarLabeller(),
+                          "NoModelLabeller": NoModelLabeller(),
+                          "mix_labellers": mix_labellers()}
```

Comment:
similar comment, in pytest, classes whose methods are tests we want to be executed need to start with `Test` (like test functions/methods need to start with `test_`) and are not initialized nor use `self`, they are a way to organize tests in order to limit the scope of a fixture (see https://docs.pytest.org/en/6.2.x/fixture.html#fixture-availability) or to group multiple tests that all have the same parametrize toghether (see https://stackoverflow.com/questions/50016862/grouping-tests-in-pytest-classes-vs-plain-functions or https://github.com/arviz-devs/xarray-einstats/blob/main/tests/test_stats.py#L126). 

I think I copied the idea in the issue from a test class in arviz so it came with the pylint disable and no self in the call signature instead of having self on the call signature but not using it in the test/function. From what I understand, the way to make some pre-prepared data available to tests is adding a method to the class decorated with the fixture thing, not initializing the test class with attributes.